app/controllers/application_controller.rb

```diff
@@ -1,5 +1,18 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
+  before_action :authenticate_user!
+
+  before_action :update_allowed_parameters, if: :devise_controller?
+
+  protected
+
+  def update_allowed_parameters
+    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :bio, :photo, :email, :password) }
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(:name, :bio, :photo, :email, :password, :current_password)
+    end
   end
 end
+
+# def current_user
+#  User.first
+# end
```

Comment:
- [ ] It's a good convention or professional practice to have our codebase clean and professional by not having commented unused code in our codebase. Kindly remove unused commented code from your codebase so your code looks clean and professional 👍🏽